partest-suite/src/test/resources/scala/tools/partest/scalajs/2.13.5/run/virtpatmat_switch.check

```diff
@@ -0,0 +1,7 @@
+zero
+one
+many
+got a
+got b
+got some letter
+scala.MatchError: 5 (of class java.lang.Byte)
```

Comment:
No new line intended?

Reply:
Ah no. I don't how that happened.

Fixed, along with some neg tests that I had not handled before.